lib/secure_headers/headers/cookie.rb

```diff
@@ -16,6 +16,11 @@ def validate_config!(config)
 
     def initialize(cookie, config)
       @raw_cookie = cookie
+      unless config == SecureHeaders::OPT_OUT
+        config ||= {}
```

Comment:
Per https://github.com/twitter/secureheaders/pull/341#issuecomment-315867482, I think we should go with

```ruby
config[:secure] = true unless config[:secure] == OPT_OUT
config[:httponly] = true unless config[:httponly] == OPT_OUT
```